convert2rhel/actions/pre_ponr_changes/kernel_modules.py

```diff
@@ -215,6 +215,15 @@ def run(self):
                     " We will continue the conversion with the following kernel modules unavailable in RHEL:\n"
                     "{kmods}\n".format(kmods="\n".join(unsupported_kmods))
                 )
+                self.add_message(
+                    level="WARNING",
+                    id="ALLOW_UNAVAILABLE_KERNEL_MODULES",
+                    message=(
+                        "Detected 'CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS' environment variable."
+                        " We will continue the conversion with the following kernel modules unavailable in RHEL:\n"
+                        "{kmods}\n".format(kmods="\n".join(unsupported_kmods))
+                    ),
+                )
```

Comment:
Can't get the lines above this (logger.warning), but will leave the suggestion anyway:

```suggestion
                message = (
                        "Detected 'CONVERT2RHEL_ALLOW_UNAVAILABLE_KMODS' environment variable."
                        " We will continue the conversion with the following kernel modules unavailable in RHEL:\n"
                        "{kmods}\n".format(kmods="\n".join(unsupported_kmods))
                    )
                logger.warning(message)
                self.add_message(
                    level="WARNING",
                    id="ALLOW_UNAVAILABLE_KERNEL_MODULES",
                    message=message,
                )
```